docs/test_plans/README.md

```diff
@@ -0,0 +1,28 @@
+# Test Plans
+
+This is a location to record all test plans that are covering celestia's ecosystem.
+
+A Test plan should be good if the following conditions are met:
+
+- Short and simple
+- Introduction
+- In-Scope
+- Out-of-Scope
+- Risks
+- Entry Conditions
+- Exit Conditions
+- Timescales
+- People
+- Test Environment
```

Comment:
Are these sections to be expected in a test plan? What does people mean as a condition?